source/playgrounds.txt

```diff
@@ -121,6 +122,25 @@ panel, the |vsce| *Overview* page, or the |vscode| Command Pallette.
 
       .. include:: /includes/steps/open-new-playground.rst
 
+   .. tab:: Your Project File
+      :tabid: project-file
+
+      1. Open a project file.
+
+         To open a file in |vscode-short|, click the file name in the
+         file explorer. 
+
+      #. Highlight the code that you want to open in a playground.
```

Comment:
Here we should be specific about "highlighting the code of _the query you want to test_"